app/controllers/application_controller.rb

```diff
@@ -27,6 +27,10 @@ def after_sign_in_path_for(_resource)
 
   private
 
+  def set_sentry_user
```

Comment:
this method isn't getting called from anywhere?

Reply:
https://github.com/IdentitySquare/IntergalacticFootball/blob/c99fabd4938735beb49390ffc2c4e5cafaaab61a/app/controllers/application_controller.rb#L11